src/lib/route-builder.js

```diff
@@ -198,7 +198,7 @@ class RouteBuilder {
       const expectedFinalAmount = new BigNumber(ilpPacket.amount).times(1 - this.slippage)
       if (expectedFinalAmount.greaterThan(nextHop.finalAmount)) {
         throw new IncomingTransferError({
-          code: 'R02',
+          code: 'R01',
```

Comment:
> Also, "Insufficient Source Amount" is R01, not R02.

In order to avoid such bugs, how about using more expressive constants instead of hard-coding the error code? For example, in `ilp-packet` we could define something along the lines `const ERROR_R01_INSUFFICIENT_SOURCE_AMOUNT = {code: ... , name: ... }`